pool/position_modify.gno

```diff
@@ -10,16 +10,16 @@ import (
 
 // modifyPosition updates a position in the pool and calculates the amount of tokens to be added or removed.
 // Returns positionInfo, amount0, amount1
-func (pool *Pool) modifyPosition(params ModifyPositionParams) (PositionInfo, *i256.Int, *i256.Int) {
+func (pool *Pool) modifyPosition(params ModifyPositionParams) (PositionInfo, *u256.Uint, *u256.Uint) {
```

Comment:
modifyPosition's return value may be able to become negative value when position is burn
we need to monitor this line if this line make the side effect.
I think it should be fine, but we have to dive.

Reply:
@notJoon 
Regarding the `transferFromAndVerify` function, the use of the test code is incorrect. Please fix it. I'm getting a uint, but the test code uses an int, which causes the test code to fail.

Reply:
https://github.com/gnoswap-labs/gnoswap/pull/403#pullrequestreview-2490796740